prov/util/src/util_mem_monitor.c

```diff
@@ -170,6 +171,16 @@ void ofi_monitor_del_cache(struct ofi_mr_cache *cache)
 	pthread_mutex_unlock(&monitor->lock);
 }
 
+/* Must be called holding monitor lock */
+bool ofi_monitor_valid(struct ofi_mem_monitor *monitor, const void *addr,
+		       size_t len)
+{
+	if (!monitor->valid)
+		return true;
```

Comment:
If a valid callback is not set by the monitor, we should set the monitor->valid() callback to a function that always returns true, rather than adding this check.  This wrapper itself isn't really needed then.

Reply:
Sounds good.